Dockerfile

```diff
@@ -22,7 +22,11 @@ RUN mkdir -p /oidc-example
 ADD keys /oidc-example/keys
 ADD static /oidc-example/static
 ADD templates /oidc-example/templates
-ADD settings.json /oidc-example/settings.json
+
+# Empty conf
+RUN echo "{}" >> /oidc-example/settings.json
```

Comment:
This snuck in. But an empty conf allows us to configure everything using environment variables in docker containers.